GiniVision/Classes/Core/GiniError.swift

```diff
@@ -26,6 +28,10 @@ public protocol GiniVisionError: Error {}
     /// Capturing could not be completed.
     case captureFailed
     
+    public var message: String {
+        // TODO: Add localized string for each case
```

Comment:
Then do it.

Reply:
As I said before I didn't want to make a lot of changes in this PR, but if it is ok for you then I will do it.